generated/nidigitalpattern/nidigitalpattern_library.cpp

```diff
@@ -3,7 +3,7 @@
 //---------------------------------------------------------------------
 // Service implementation for the NI-Digital Pattern Driver Metadata
 //---------------------------------------------------------------------
-#include "nidigital_library.h"
+#include "nidigitalpattern_library.h"
 
 #if defined(_MSC_VER)
 static const char* kLibraryName = "niDigital_64.dll";
```

Comment:
@jonlugoatx should namespace on line 14 below be updated to `nidigitalpattern_grpc` as well?

Reply:
Yes, we should keep the module name and namespace consistent

Reply:
Sure. I have made changed the namespace name and made changes at appropriate location.